### PR TITLE
Fix mobile status bar issues

### PR DESCRIPTION
**Pull request by kadisonm**
Hopefully fixes all this. I tested on pc and mobile and it seems to work pretty well for me.
- Fixes status bar showing over navmenu on mobile by moving it up #124.
- Also fixes status bar not hiding new elements that are added to it with only show last synced on #124.
- Fixes the default settings. Now on computer status bar will default to on but only show last synced off. While on mobile status bar will default to off, but show only last synced defaults to true.

To explain what I did. I created a Mutation Observer that watches the status bar element for any children added and reloading the status bar to hide them when there are. The reason this has its own toggle function is so that the observer can be disabled when show only last synced is off, since it's not needed then.

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -1211,6 +1236,39 @@ export default class RemotelySavePlugin extends Plugin {
...
+        let byPlugin = false;
+
+        for (const mutation of mutationList) {
+          if (mutation.type === "childList" && mutation.addedNodes.length > 0) {
+            shouldCall = true;
+          }
+
+          mutation.addedNodes.forEach((node) => {
+            if ((node as Element).className === "status-bar-item plugin-remotely-secure") {
+              byPlugin = true;
```
Comment (kadisonm) [inline src/main.ts:1257]: This is quite confusing to look at but, when the status bar element is added with toggleStatusBar it triggers the observer's callback. This is essentially just checking that the new element wasn't created by Remotely Sync. It also only calls toggleStatusBar if an element is added and ignores elements being removed.

Comment (sboesen) [reply src/main.ts:1257]: Appreciate you clarifying, thank you!

**Comment by sboesen** [comment]
LGTM, thank you for the fix & sorry for the delay merging!

**Comment by kadisonm** [comment]
No worries!

**Pull request merged**